### DISCUSSION TITLE
Quoted output style for properties format

### DISCUSSION THREAD

**Discussion by mkhlopot** [mikefarah/yq#1022] [Q&A]
I'm trying to get yq to output my values as properties with quoted values, i.e.
`section.value="abc"`
I\m trying `yq eval '(. style="double"| to_props)' values.yaml` with different placing for style tag, but all I'm getting is still unquoted output:
`section.value=abc`

**Comment by mikefarah** [comment]
The `style` operator only really works for `yaml` output, as neither the json nor properties encoder support field styles.

The only way I can think of is to wrap all the string values with quotes:

```
yq e -o props '(.. |  select(tag == "!!str")) |= ("\"" + . + "\"")' my_file.yaml
```

Another thing to keep mind of is `to_props` as an operator in the expression vs `-o props` as a yq argument.

`to_props` is meant to be used for embedding property formatted strings into the yaml document (e.g. `.a.b = (.c | to_props)` - whereas `-o props` is for converting the entire output to properties format. Though I suppose if you run `to_props` at the top level there's not much difference 🤔
